cmd/maya-apiserver/cstor-operator/controller.go

```diff
@@ -138,45 +138,46 @@ func (cb *ControllerBuilder) Build() (*Controller, error) {
 	return cb.Controller, nil
 }
 
-// addSpc is the add event handler for spc.
+// addSpc is the add event handler for cspc.
```

Comment:
nit: can we rename it for better understanding in logs.

Reply:
This is what we have followed in controller. Even if we want to do something wrt naming in controller  ,will take it later sometimes.
BTW, current name  looks intuitive and concise to me.